Modules/ThirdParty/HDF5/src/itkhdf5/CMakeLists.txt

```diff
@@ -785,7 +785,13 @@ set (HDF5_SRC_INCLUDE_DIRS
     ${HDF5_SRC_INCLUDE_DIRS}
     ${H5FD_SUBFILING_DIR}
 )
+
+if (FALSE) # XXX(kitware): Disable subfilling because it requires parallel.
```

Comment:
I think that `#[[ITK --start` is used throughout this file.

This commit should probably not be squashed. Its message should be updated.

Reply:
> I think that `#[[ITK --start` is used throughout this file.

Actually no: https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/ThirdParty/HDF5/src/itkhdf5/CMakeLists.txt